### PR TITLE
Close scanners, remove stuff, she nae nae gains

### PR DESCRIPTION
**Pull request by john-kick**

### CODE REVIEW

File: src/sud/aufgaben/occupationalTherapy/form/Triangle.java
Comment (shyrack) [inline src/sud/aufgaben/occupationalTherapy/form/Triangle.java:1]: No, such a horrible name cannot be supported by this open source project. Screw you!

File: src/sud/aufgaben/occupationalTherapy/form/Triangle.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public static boolean isTriangle(double a, double b, double c) {
     }
 
     public static void throwInvalidTriangleException() {
-        throw new IllegalArgumentException("Andrè ist ein Dreieck, du Wicht, aber diese Werte sind es nicht.");
```
Comment (shyrack) [inline src/sud/aufgaben/occupationalTherapy/form/Triangle.java:73]: Dis has to be corrected

File: src/sud/aufgaben/occupationalTherapy/form/Triangle.java
Comment (john-kick) [reply src/sud/aufgaben/occupationalTherapy/form/Triangle.java:1]: Neuken deen mamer in de keukeun

**Review by shyrack — CHANGES_REQUESTED** [review]

File: src/sud/aufgaben/occupationalTherapy/form/Triangle.java (hunk 1 above)
Comment (john-kick) [reply src/sud/aufgaben/occupationalTherapy/form/Triangle.java:73]: COPIUM

**Pull request merged**